src/Http/Http.Extensions/src/RequestDelegateFactory.cs

```diff
@@ -582,6 +659,50 @@ private static Expression BindParameterFromValue(ParameterInfo parameter, Expres
                         valueExpression,
                         Expression.Convert(Expression.Constant(parameter.DefaultValue), parameter.ParameterType)));
             }
+            else if (parameter.ParameterType == typeof(IFormFile))
```

Comment:
Again, copy-pasta from the above block for `typeof(string)`, might be redundant with the change I made to `GetValueFromProperty()` to allow the return type to be different, or the implementation might want to be something completely different.

Reply:
Yep - updated in a9ac8a09ceb3562e68ba424760a408f05596b836.